bin/runtime-common/src/messages.rs

```diff
@@ -186,16 +170,16 @@ pub trait BridgedChainWithMessages: ChainWithMessages {
 pub type ThisChain<B> = <B as MessageBridge>::ThisChain;
 /// Bridged chain in context of message bridge.
 pub type BridgedChain<B> = <B as MessageBridge>::BridgedChain;
+/// Underlying chain type.
+pub type UnderlyingChainOf<C> = <C as ChainWithMessages>::Chain;
```

Comment:
@svyatonik 
just very nit: what about renaming `ChainWithMessages` -> `UnderlyingChain`,
`ChainWithMessages` here is not very descriptive and also we have two `trait ChainWithMessages`